### PR TITLE
Add link to ELTAP Fabrics in Footer and MainNavigation

### PR DESCRIPTION
**Pull request by PiotrZaranek**

### CODE REVIEW

File: be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.de.resx [hunk 1]
```diff
@@ -708,4 +708,7 @@
   <data name="Sku" xml:space="preserve">
     <value>Sku</value>
   </data>
+  <data name="EltapFabrics" xml:space="preserve">
+    <value>ELTAP-Stoffe</value>
```
Comment (SebaKas21) [inline be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.de.resx]: **suggestion (non-blocking):** Remove - and add space between ELTAP and Stoffe

File: be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/Footers/FooterModelBuilder.cs [hunk 2]
```diff
@@ -29,6 +29,12 @@ public FooterViewModel BuildModel()
         {
             var links = new List<LinkViewModel>
             {
+                new LinkViewModel
```
Comment (SebaKas21) [inline be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/Footers/FooterModelBuilder.cs]: **suggestion:** Please change this to _

File: be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/Footers/FooterModelBuilder.cs [hunk 3]
```diff
@@ -29,6 +29,12 @@ public FooterViewModel BuildModel()
...
+                {
+                    Text = this.globalLocalizer["EltapFabrics"],
```
Comment (SebaKas21) [inline be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/Footers/FooterModelBuilder.cs]: **suggestion:** Move it off the list and check if the url is set, beacuse ADRK does not have a website with fabrics

File: be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/MainNavigations/MainNavigationModelBuilder.cs [hunk 4]
```diff
@@ -36,56 +36,50 @@ public async Task<MainNavigationViewModel> BuildModelAsync(ComponentModelBase co
...
+                {
+                    Text = this.globalLocalizer.GetString("AvailableProducts"),
+                    Url = this.linkGenerator.GetPathByAction("Index", "AvailableProducts", new { Area = "Products", culture = CultureInfo.CurrentUICulture.Name })
+                },
+                new LinkViewModel
+                {
+                    Text = this.globalLocalizer.GetString("Outlet").Value,
+                    Url = this.linkGenerator.GetPathByAction("Index", "Outlet", new { Area = "Products", culture = CultureInfo.CurrentUICulture.Name })
+                },
+                new LinkViewModel
```
Comment (SebaKas21) [inline be/src/Project/Web/Buyer/Buyer.Web/Shared/ModelBuilders/MainNavigations/MainNavigationModelBuilder.cs:59]: **suggestion:** check the link here as in the footer

File: misc/deployment/BuyerWeb/helm-chart/templates/deployment.yaml [hunk 5]
```diff
@@ -60,6 +60,8 @@ spec:
           value: {{ .Values.downloadCenterUrl }}
         - name: MakeComplaintUrl
           value: {{ .Values.makeComplaintUrl }}
+        - name: FabricsWebUrl
+          value: {{ .Values.makeComplaintUrl }}
```
Comment (SebaKas21) [inline misc/deployment/BuyerWeb/helm-chart/templates/deployment.yaml]: **suggestion:** Copy & Paste | Set fabricsWebUrl in value

**Review by SebaKas21 — CHANGES_REQUESTED** [review]

**Comment by dawiddworak88** [comment]
Hi @PiotrZaranek, Hi @SebaKas21, I have to reject this PR because it's making the solution ELTAP specific which is wrong. It will create issues when I deploy it for ADRK. Please consult these kind of initiatives with me before implementation. We'll come back to it when the redesigned footer with Stripe is deployed to production.

**Pull request closed without merging**